crates/ra_hir_expand/src/name.rs

```diff
@@ -152,6 +152,8 @@ pub const RANGE_INCLUSIVE_TYPE: Name = Name::new_inline_ascii(b"RangeInclusive")
 pub const RANGE_TO_INCLUSIVE_TYPE: Name = Name::new_inline_ascii(b"RangeToInclusive");
 pub const RANGE_TO_TYPE: Name = Name::new_inline_ascii(b"RangeTo");
 pub const RANGE_TYPE: Name = Name::new_inline_ascii(b"Range");
+pub const NEG_TYPE: Name = Name::new_inline_ascii(b"Neg");
+pub const NOT_TYPE: Name = Name::new_inline_ascii(b"Not");
```

Comment:
These should be `_TRAIT`, not  `_TYPE`.

Reply:
Both Future and IntoIterator are _TYPE so was just following the existing naming. Can still change it if neede of course :)

Reply:
Ah, ok, so `FUTURE` is `future` and `FUTURE_TYPE` is `Future`. But `DEFAULT_TRAIT` is trait. I guess it's better to leave as is for the time being, and then just add an `N!` macro, such that `N![Not]` is unambigious.